app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -6,11 +6,14 @@ description: |-
 related_content:
     Explore the benefits of teaching: "/life-as-a-teacher/pay-and-benefits"
     If you've worked as an unqualified teacher: "/train-to-be-a-teacher/assessment-only-route-to-qts"
-external_content:
-    - content/train-to-be-a-teacher/promos/find-your-course
 cta_adviser:
   adviser:
     text: "An adviser with years of teaching experience can answer all your questions about salaried teacher training. Chat by phone, text or email, as little or as often as you need."
+calls_to_action:
+    find:
```

Comment:
Indenting is a little off here, `find` (and everything under it) should be closer to `calls_to_action`, like how `adviser` lines up with `cta_adviser`